content/quickstart.md

```diff
@@ -48,43 +123,31 @@ $ minishift config set memory 8384
 $ minishift config set cpus 2
 ```
 
-### Template selection
-
 Deploying Syndesis is made easy thanks to [OpenShift templates](https://docs.openshift.org/latest/dev_guide/templates.html). The template to use in the installation instructions depend on your use case:
 
-* **Developer** : Use the template [`syndesis-dev-restricted`](https://raw.githubusercontent.com/syndesisio/syndesis/master/app/deploy/syndesis-dev-restricted.yml) which directly references Docker images without image streams. Then before building the images e.g. with `mvn fabric8:build` set your `DOCKER_HOST` environment variable to use the Minishift Docker daemon via `eval $(minishift docker-env)`. When new images are built you only need to delete the appropriate pod so that the new pod spinning up will use the freshly built image.
+* **User** : In case you only want to have the latest version of Syndesis on your local Minishift installation, use the template [`syndesis`](https://raw.githubusercontent.com/syndesisio/syndesis/master/app/deploy/syndesis.yml) which uses image stream referring to the published Docker Hub images. Minishift will update its images and trigger a redeployment when the images at Docker Hub changes. Therefore it checks every 15 minutes for a changed image. You do not have to do anything to get your application updated, except for waiting on Minishift to pick up new images.
```

Comment:
"Therefore," or "As such," although of course this is not an English essay in college, I'm not sure why I'm correcting this. 😂  It's really perfect as is.

Reply:
Well, my Grammarly checker allows both. I'm very lousy in english grammar and commas, so I typically trust it, but lets update here :-)